tests/src/Unit/OgAccessEntityTestBase.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\og\Unit\OgAccessEntityTestBase.
+ */
+
+namespace Drupal\Tests\og\Unit;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Entity\EntityManagerInterface;
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+
+class OgAccessEntityTestBase extends OgAccessTestBase {
+
+  protected $entity;
+
+  public function setup() {
+    parent::setUp();
+    if (!defined('OG_STATE_ACTIVE')) {
+      define('OG_STATE_ACTIVE', 1);
+    }
+
+    $field_definition = $this->prophesize(FieldDefinitionInterface::class);
+    $field_definition->getType()->willReturn('og_membership_reference');
+    $field_definition->getFieldStorageDefinition()
+      ->willReturn($this->prophesize(FieldStorageDefinitionInterface::class)->reveal());
+    $field_definition->getSetting("handler_settings")->willReturn([]);
+    $field_definition->getName()->willReturn($this->randomMachineName());
+
+    $entity_type_id = $this->randomMachineName();
+    $bundle = $this->randomMachineName();
+    $entity_id = mt_rand(20, 30);
+
+    $entity_type = $this->prophesize(EntityTypeInterface::class);
+    $entity_type->getListCacheTags()->willReturn([]);
```

Comment:
But otherwise yes -- note that's a dangerous road to go down because you might a write test with the wrong expectations so be sure to sanity check when doing it that way.